airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_variables.py

```diff
@@ -728,7 +709,10 @@ class TestBulkVariables(TestVariableEndpoint):
                     "actions": [
                         {
                             "action": "create",
-                            "entities": [{"key": "new_var1", "value": "new_value1"}],
+                            "entities": [
+                                {"key": "new_var1", "value": "new_value1"},
+                                {"key": "new_var2", "value": ["new_value1"]},
```

Comment:
Can you add a few more test cases to cover the "standard" range of JSON, using `int`, `null`, `dict` as well?